web/src/components/atoms/InnerContents/complex.tsx

```diff
@@ -24,7 +24,7 @@ export default ComplexInnerContents;
 const PaddedContent = styled.div`
   display: flex;
   margin: 16px 0 0 16px;
-  height: calc(100vh - 66px);
+  min-height: calc(100vh - 66px);
```

Comment:
Please keep it height, and add overflow: auto; to the PaddContent. If you check test environment, if you make the height short, you'll notice the right fields list and the list of models scroll by themselves. With your fix here it breaks that and EVERYTHING scrolls together, which isn't great.